sorbet/rails-rbi/models/collection/habtm_managers.rbi

```diff
@@ -77,7 +57,6 @@ module Collection::HABTM_Managers::CustomFinderMethods
 end
 
 class Collection::HABTM_Managers < ActiveRecord::Base
-  include Collection::HABTM_Managers::GeneratedAttributeMethods
```

Comment:
Did you run `bundle exec srb rbi hidden-definitions`?   I wonder why this disappeared?

Reply:
Nope.  Same comment: https://github.com/sul-dlss/happy-heron/pull/644#discussion_r535454919

I did do a rebase from  master (and on #644 too)